### DISCUSSION TITLE
Manage my presence status (Away, unavailable, available..)

### DISCUSSION THREAD

**Discussion by jlemangarin** [element-hq/element-meta#463] [Ideas]
**Is your suggestion related to a problem? Please describe.**
I found this issue (https://github.com/vector-im/element-web/issues/4199) which seems to be the presence feature but I can't find references for the possibility of changing it's own presence to indicate we are unavailable for exemple. This is wanted by our customers (https://community.ceo-vision.com/topic/482/statut-sur-le-tchat)

**Describe the solution you'd like**
Possibility to change our own presence in our profile menu dropdown

**Describe alternatives you've considered**
//

**Additional context**
//

**Comment by ptman** [comment]
IIRC setting presence was available previously. Before the redesign? This should really be brought back. And the custom status labs doesn't do anything?

**Comment by HarHarLinks** [comment]
in addition the mentioned, element should also support invisible/appear offline status. i.e. don't send presence (and read markers...) at all.

**Comment by aaronraimist** [comment]
Related: https://github.com/vector-im/element-web/issues/4706

**Comment by a-0-dev** [comment]
This seems to be the most recent thread about this topic, the issues relating to this are older...
I would like to emphasize this is actually quite a big deal for many users, e.g. in my institution, which is holding back from using element (and sadly, often, thus matrix!) for their job. They feel uncomfortable knowing others see whether you are "online" during work hours.

It seems to be an easy fix to me, adding one toggle in the settings which suppresses sending presence events, but I don't know JavaScript so I'm afraid I cannot help with this.

**Comment by aaronraimist** [comment]
@a-0-dev If your entire institution feels that way, you can disable presence for your homeserver.

**Comment by a-0-dev** [comment]
@aaronraimist We considered this, however it's too important as a feature to many users to do so, having this option for more privacy-oriented users is definitely a better fitting solution.

**Comment by ddmesh** [comment]
I have just found that issue (online status) in roadmap and added my comment there:
https://github.com/vector-im/roadmap/issues/22#issuecomment-1307025517

Just to have this as reference in this discussion:
~~~
Long time since last update on this issue. It is really important to allow people to hide their online status. currently all messengers do support hiding the online status.
I have seen that there is an api call "set_presence" which allows this. but this such a configuration should be available in client.
As users may install different clients, a server side (home server) configuration for the whole account which controls the
online status, would be great.

In LABs flags there is no flag that allows to being "offline", only the read receipt can be muted.
Ar there any plans to set this issue to a higher priority?

Thanks a lot
~~~

I agree totally the privacy is the most important thing and every user should have the possibility to hide their online status, like other messengers can (not just the read status).
